android/build.gradle

```diff
@@ -42,7 +42,7 @@ android {
 
 dependencies {
     testImplementation 'junit:junit:4.13.2'
-    implementation 'com.clevertap.android:clevertap-android-sdk:4.6.3'
-    implementation 'androidx.fragment:fragment:1.3.6'
-    implementation 'androidx.core:core:1.3.0'
+    api 'com.clevertap.android:clevertap-android-sdk:4.6.3'
```

Comment:
@shivamsharma2710 please update android sdk to v4.6.6

Reply:
This is resolved. Please check.